### DISCUSSION TITLE
Does linkerd support WC3 Propagation headers

### DISCUSSION THREAD

**Discussion by jseiser** [linkerd/linkerd2#11813] [General]
I can see this merge request went in, https://github.com/linkerd/linkerd2-proxy/pull/2179

but the docs still only indicate B3.

https://linkerd.io/2.14/tasks/distributed-tracing/#modify-the-application
https://linkerd.io/2.14/tasks/distributed-tracing/#i-dont-see-any-spans-for-the-proxies

**Comment by wmorgan** [comment]
Sorry for the long delay! I believe the answer is yes, and the docs are just a bit behind.
